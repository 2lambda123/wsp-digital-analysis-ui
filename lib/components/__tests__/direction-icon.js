// @flow
import {shallow} from 'enzyme'
import toJSON from 'enzyme-to-json'
import React from 'react'

import DirectionIcon from '../direction-icon'

describe('Component > DirectionIcon', () => {
  it('renders correctly', () => {
    const tree = shallow(
      <DirectionIcon
        bearing={123}
        clickable
        color='blue'
        coordinates={[12, 34]}
        iconSize={20}
      />
    )
    expect(toJSON(tree)).toMatchSnapshot()
  })
})
