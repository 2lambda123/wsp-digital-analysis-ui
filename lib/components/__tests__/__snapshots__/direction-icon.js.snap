// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component > DirectionIcon renders correctly 1`] = `
<div
  className={undefined}
  id={undefined}
  style={undefined}
>
  <g
    data-props={
      Object {
        "clickable": true,
        "icon": LeafletDirectionIcon {
          "callInitHooks": [Function],
          "createIcon": [Function],
          "createShadow": [Function],
        },
        "position": Array [
          34,
          12,
        ],
      }
    }
    type="Marker"
  />
</div>
`;
